faunadb/errors.py

```diff
@@ -18,6 +20,29 @@ class FaunaError(Exception):
   For documentation of error types, see the `docs <https://faunadb.com/documentation#errors>`__.
   """
 
+  @staticmethod
+  def from_status_code(code, response_dict):
+    # pylint: disable=no-member
+    if 200 <= code <= 299:
```

Comment:
We have this 2 lines of code in multiple places. We should consolidate it. I think doing here that this line becomes an if not then raise at the end of this if elif block. If it's a valid code, do nothing, so we can call something like this:

```
# Make the request
response = do_request()
# raise for error
FaunaError.from_response(response)
# parse the response
# ...
```

(doesn't have to look exactly like that, just some pseudocode)